Regression/Checksum/benchmarks_json/test_3d_open_bc_poisson_solver_heffte.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I thought so too, but there isn't one in development right now.
Something must have happened when we moved to `ctest`, maybe?
Anyway, the main problem here is that the `heffte` test never runs in CI, so that's why we didn't notice.

Reply:
Actually, from [here](https://github.com/ECP-WarpX/WarpX/pull/4937/files#diff-9d054a23083a17396f4b4a929f708e44643af3b2e24b492b15e1f2047138426a) it seems like that the checksums were saved in the wrong file.